components/org.wso2.carbon.identity.keyrotation/src/main/java/org.wso2.carbon.identity.keyrotation/config/FileBasedKeyRotationConfigProvider.java

```diff
@@ -79,6 +79,9 @@ private void initConfig(Properties properties) throws KeyRotationException {
             logger.log(Level.WARN, "Not a valid number. Falling back to default chunk size.", e);
             keyRotationConfig.setChunkSize(DBConstants.DEFAULT_CHUNK_SIZE);
         }
+        String enableWorkflowMigrator =
+                StringUtils.isNotBlank(properties.getProperty(KeyRotationConstants.ENABLE_WORKFLOW_MIGRATOR)) ?
+                properties.getProperty(KeyRotationConstants.ENABLE_WORKFLOW_MIGRATOR) : "true";
```

Comment:
Default value needs to be discussed.

Reply:
I assume this tool is generic for any product version. If so, is there a way for us to identify the product version from the tool?

Reply:
Identifying the product version won't be of much use here because this will need to be enabled even in IS 7 if the workflow connector is being used.